styles.css

```
@@ -0,0 +1,156 @@
+* {
+    box-sizing: border-box;
+}
+
+.u-list-reset {
+    list-style: none;
+    margin-bottom: 0;
+    margin-top: 0;
+    padding-left: 0;
+}
+
+.u-list-inline li {
+    display: inline-block;
+    margin-left: -9px;
+    margin-right: 15px;
+    margin-top: 15px;
+}
+
+/* ////// base elements /////// 
+//////////////////////////////*/
+
+body {
+    background-color: rgb(244, 136, 90);
+    color: black;
+    font-family: 'Libre Baskerville', serif;
+    font-size: 7px;
+    line-height: 10px;
+}
+
+
+article {
+    background-color: rgb(255, 255, 255);
+    border-radius: 10px;
+    height: 196px;
+    margin: 100px 50px;
+    width: 436px;
+}
+
+img {
+    border-bottom-left-radius: 5%;
+    border-top-left-radius: 5%;
+    max-width: 100%;
+    object-fit: cover;
+}
+
+h2 {
+    color: black;
+    font-family: 'Montserrat', sans-serif;
+    font-size: 12px;
+    line-height: 14px;
+    margin-top: 20px;
+}
+
+h3 {
+    color: rgb(142, 167, 66);
+    font-family: 'Montserrat', sans-serif;
+    font-size: 8px;
+    text-decoration-line: none;
+    text-transform: uppercase;
+}
+
+/* icons of bookmark, star, share */
+ul {
+    position: absolute;
+    right: 0;
+    top: 0;
```

Instead of mucking about with the individual margins on the `li` elements in this list, you can instead update the values for right and top here:

```suggestion
    position: absolute;
    right: 10px;
    top: 10px;
```